CRM/Utils/Request.php

```diff
@@ -133,6 +133,16 @@ protected static function getValue($name, $method) {
     foreach (($method ?? []) as $key => $value) {
       if (strpos($key, 'amp;') !== FALSE) {
         $method[str_replace('amp;', '', $key)] = $method[$key];
+        if ($method === '$_POST' && $_SERVER['CONTENT_TYPE'] === 'application/json') {
+          static $post = NULL;
+          if (!isset($post)) {
+            $rawPost = file_get_contents('php://input');
+            $post = json_decode($rawPost, TRUE) ?? [];
```

Comment:
Does this give reasonable performance? The input JSON is decoded every time a value is requested, instead of only once per request.

Reply:
In practice, I don't think this is called very many times per request.

Reply:
@Sjord Shouldn't the `static` keyword on line 137 (`static $post = NULL`) prevent this from being decoded multiple times?

Reply:
Yes, you are right. Thanks for pointing that out. So it *is* decoded only once per request, I was mistaken about that.